Mono.Cecil.props

```diff
@@ -31,6 +31,22 @@
     <WarningLevel>4</WarningLevel>
     <RunCodeAnalysis>false</RunCodeAnalysis>
   </PropertyGroup>
+  <PropertyGroup Condition=" $(Configuration.EndsWith('Debug_ReadOnly')) ">
```

Comment:
Maybe change the previous property group condition to Contains('Debug') and Contains('Release') and add a property group Contains('ReadOnly') to avoid duplication?